staging/src/k8s.io/apiserver/pkg/authorization/authorizer/interfaces.go

```diff
@@ -19,6 +19,7 @@ package authorizer
 import (
 	"net/http"
 
+	authorization "k8s.io/api/authorization/v1"
```

Comment:
Sorry let me clarify on this. My original comment was that importing API types from this package is a big change since this package historically hasn't relied on those definitions.

I think the internal types are the correct API types to use (`"k8s.io/kubernetes/api/authorization"`), but I want to get sign off from someone in @kubernetes/sig-auth-pr-reviews that adding a dependency on API types is okay.